tools/pipeline-generator/Azure.Sdk.Tools.PipelineGenerator/Conventions/IntegrationTestingPipelineConvention.cs

```diff
@@ -18,7 +18,10 @@ public IntegrationTestingPipelineConvention(ILogger logger, PipelineGenerationCo
 
         public override string GetDefinitionName(SdkComponent component)
         {
-            return component.Variant == null ? $"{Context.Prefix} - {component.Name} - tests" : $"{Context.Prefix} - {component.Name} - tests.{component.Variant}";
+            var baseName = component.Variant == null
```

Comment:
Looks like this is now duplicated everywhere. Perhaps we should make this the default implementation from the base class and then all the override can add their postfix at the end.